src/main/java/net/smartcosmos/dto/relationships/RelationshipLookupSpecific.java

```diff
@@ -0,0 +1,41 @@
+package net.smartcosmos.dto.relationships;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Setter;
+
+import java.beans.ConstructorProperties;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+public class RelationshipLookupSpecific {
+    @Setter(AccessLevel.NONE)
+    private String version = "1";
+
+    private String entityReferenceType;
+    private String referenceUrn;
+    private String type;
+    private String relatedEntityReferenceType;
+    private String relatedReferenceUrn;
+
+    @Builder
+    @ConstructorProperties({"entityReferenceType", "referenceUrn", "type",
+        "relatedEntityReferenceType", "relatedReferenceUrn"})
+    public RelationshipLookupSpecific(
+        String entityReferenceType,
+        String referenceUrn,
+        String type,
+        String relatedEntityReferenceType,
+        String relatedReferenceUrn)
+    {
+        this.entityReferenceType = entityReferenceType;
+        this.referenceUrn = referenceUrn;
+        this.type = type;
+        this.relatedEntityReferenceType = relatedEntityReferenceType;
+        this.relatedReferenceUrn = relatedReferenceUrn;
+    }
```

Comment:
Add `this.version = 1` here (without adding it to the argument list -- or use both field initialization and the constructor. In this case, I'd suggest to use a constant for the actual version: `private static final int VERSION = 1` and then set `this.version = VERSION`.